README.md

```diff
@@ -0,0 +1,342 @@
+```
+
+Edit `~/brigade-slack-gateway-values.yaml`, making the following changes:
+
+* `brigade.apiAddress`: Address of the Brigade API server, beginning with
+  `https://`
+
+* `brigade.apiToken`: Service account token from step 2
+
+* `github.apps`: Specify the details of your Slack App(s), including:
+
+    * `appID`: This is the unique ID for a Slack App. You took note of this
+      value in a previous step.
+
+    * `appSigningSecret`: This secret can be used to validate signatures on
+      inbound slash commands. You took note of this value in a previous step.
+
+    * `apiToken`: This token can be used to authenticate when sending messages
+      back to Slack. You took note of this value in a previous step.
+
+* `receiver.host`: Set this to the host name where you'd like the gateway to be
+  accessible.
+
+Save your changes to `~/brigade-slack-gateway-values.yaml` and use the following
+command to install the gateway using the above customizations:
+
+```console
+$ helm install brigade-slack-gateway \
+    oci://ghcr.io/brigadecore/brigade-slack-gateway \
+    --version v0.1.0 \
+    --create-namespace \
+    --namespace brigade-slack-gateway \
+    --values ~/brigade-slack-gateway-values.yaml
+```
+
+### 4. (RECOMMENDED) Create a DNS Entry
+
+If you installed the gateway without enabling support for an ingress controller,
+this command should help you find the gateway's public IP address:
+
+```console
+$ kubectl get svc brigade-slack-gateway-receiver \
+    --namespace brigade-slack-gateway \
+    --output jsonpath='{.status.loadBalancer.ingress[0].ip}'
+```
+
+If you overrode defaults and enabled support for an ingress controller, you
+probably know what you're doing well enough to track down the correct IP without
+our help. 😉
+
+With this public IP in hand, edit your name servers and add an `A` record
+pointing your domain to the public IP.
+
+__Note:__ If you do not want to use a domain or subdomain name, or are unable
+to, and elected to use a placeholder URL when initially setting up your Slack
+App, return to https://api.slack.com/apps and edit your App's configuration
+to send your slash command to `https://<public ip>/slash-commands`.
+
+### 5. Add a Brigade Project
+
+You can create any number of Brigade projects (or modify an existing one) to
+listen for events that were sent from Slack to your gateway and, in turn,
+emitted into Brigade's event bus. You can subscribe to all event types emitted
+by the gateway, or just specific ones.
+
+In the example project definition below, we subscribe to all events emitted by
+the gateway, provided they've originated from the fictitious Slack app with the
+ID `FAKEAPPID` (see the `appID` qualifier).
+
+```yaml
+apiVersion: brigade.sh/v2-beta
+kind: Project
+metadata:
+  id: slack-demo
+description: A project that demonstrates integration with Slack
+spec:
+  eventSubscriptions:
+  - source: brigade.sh/slack
+    qualifiers:
+      appID: FAKEAPPID
+    types:
+    - *
```

Comment:
```suggestion
    - "*"
```

Reply:
I think it worked for me as is. I might be wrong. Either way, I agree your suggestion is clear and safer.